### ISSUE TITLE
Logging section json prettify and custom keyword coloring

### ISSUE THREAD

**Issue by RobinBaeckman** [lensapp/lens#3045]
**What would you like to be added**:
Formatting/prettifying json in logging section and add so keywords can be color coded.

**Why is this needed**:
Its difficult to read the logs when they are on one line, it would be nice to have the option to "prettify" json log lines. To improve readability further have a section where user can input keywords correlated to a color so logs can be both prettified and have keyword coloring 

- Kubernetes distribution: GKE
- Desktop OS: macOS

**Comment by mjpowersjr** [comment]
I agree that reading structured json logs is not so great. Maybe a more flexible approach would be for Lens to support configuring a 'log filter' that could point to any cmd line tool. For example, we use a tool called `pino-pretty` to format our JSON logs for humans. This approach would allow people to customize the log formatting.

**Comment by RobinBaeckman** [comment]
that would be great @mjpowersjr

**Comment by backtozero** [comment]
Waiting for this

**Comment by foal** [comment]
What about wrapping something like this (https://github.com/gistia/json-log-viewer) as Lens Extensions to provide an alternative log viewer?

**Comment by maxiangelo** [comment]
this would be great
